src/applications/static-pages/cta-widget/components/messages/Verify.jsx

```diff
@@ -1,36 +1,75 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+
+import {
+  CSP_IDS,
+  SERVICE_PROVIDERS,
+} from '~/platform/user/authentication/constants';
+import {
+  VerifyIdmeButton,
+  VerifyLogingovButton,
+} from '~/platform/user/authentication/components/VerifyButton';
 import CallToActionAlert from '../CallToActionAlert';
 
-const Verify = ({ headerLevel, primaryButtonHandler, serviceDescription }) => {
+/**
+ * Alert to show a user that is not verified.
+ * @property {string} signInService user's signin service name
+ * @property {string} headerLevel optional heading level
+ * @property {string} serviceDescription optional description of the service that requires verification
+ */
+const Verify = ({ signInService, headerLevel, serviceDescription }) => {
+  const headingPrefix = 'Verify your identity';
+  const headline = serviceDescription
+    ? `${headingPrefix} to ${serviceDescription}`
+    : headingPrefix;
+  const serviceProviderLabel = SERVICE_PROVIDERS[signInService].label;
+
+  const singleVerifyButton =
+    signInService === CSP_IDS.LOGIN_GOV ? (
+      <VerifyLogingovButton />
+    ) : (
+      <VerifyIdmeButton />
+    );
+
   const content = {
-    heading: `Please verify your identity to ${serviceDescription}`,
+    heading: headline,
     headerLevel,
     alertText: (
       <>
         <p>
-          We need to make sure you’re you — and not someone pretending to be you
-          — before we can give you access to your personal and health-related
-          information. This helps us keep your information safe and prevent
-          fraud and identity theft.
+          We need you to verify your identity for your{' '}
+          <strong>{serviceProviderLabel}</strong> account. This step helps us
+          protect all Veterans’ information and prevent scammers from stealing
+          your benefits.
         </p>
         <p>
-          <strong>This one-time process takes about 5 to 10 minutes.</strong>
+          This one-time process often takes about 10 minutes. You’ll need to
+          provide certain personal information and identification.
+        </p>
+        <p>{singleVerifyButton}</p>
+        <p>
+          <va-link
+            href="/resources/verifying-your-identity-on-vagov/"
+            text="Learn more about verifying your identity"
+            disableAnalytics
+          />
         </p>
       </>
     ),
-    primaryButtonText: 'Verify your identity',
-    primaryButtonHandler,
     status: 'warning',
   };
 
   return <CallToActionAlert {...content} />;
 };
 
+Verify.defaultProps = {
+  signInService: CSP_IDS.ID_ME,
+};
+
 Verify.propTypes = {
-  primaryButtonHandler: PropTypes.func.isRequired,
-  serviceDescription: PropTypes.string.isRequired,
-  headerLevel: PropTypes.string,
+  headerLevel: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  serviceDescription: PropTypes.string,
+  signInService: PropTypes.oneOf([CSP_IDS.ID_ME, CSP_IDS.LOGIN_GOV]),
```

Comment:
I'm wondering if it is possible for this component to be called with other `CSP_IDS` and what the result would be.

Reply:
After reading the `getVerifyAlert` function in `cta-widget/index.js`, the above shouldn't be a concern.